ngraph/frontend/paddlepaddle/src/op/batch_norm.cpp

```diff
@@ -32,15 +20,42 @@ namespace ngraph
                     auto beta = node.get_ng_input("Bias");
                     auto mean = node.get_ng_input("Mean");
                     auto variance = node.get_ng_input("Variance");
-                    return node.default_single_output_mapping(
-                        {std::make_shared<ngraph::opset6::BatchNormInference>(
-                            data,
+                    auto data_layout = node.get_attribute<std::string>("data_layout");
+
+                    PDPD_ASSERT((data_layout == "NCHW" || data_layout == "NHWC"),
```

Comment:
@nosovmik As I know PDPD depends on `ngraph`, why cannot we use `NGRAPH_CHECK`?

Reply:
we actually are going to split this macro to several ones FRONT_END_GENERAL_CHECK, PDPD_OP_VALIDATION_CHECK, etc.
See PR #36 for details. This will provide more convenient exception handling on python side